dotCMS/src/integration-test/java/com/dotcms/content/elasticsearch/business/ESMappingAPITest.java

```diff
@@ -145,6 +148,67 @@ public void test_toMap_fileasset_txt_shouldSuccess() throws Exception {
 
     }
 
+    @Test
+    public void test_toMap_when_textFieldShouldNotBeIncluded(){
+        final ESMappingAPIImpl esMappingAPI = new ESMappingAPIImpl();
+        Config.setProperty("CREATE_TEXT_INDEX_FIELD_FOR_NON_TEXT_FIELDS", false);
+
+        final Map<String, Object> contentletMap = esMappingAPI.toMap(TestDataUtils
+                .getNewsContent(true, language.getId(),
+                        TestDataUtils.getNewsLikeContentType().id()));
```

Comment:
Codacy found an issue: [Unnecessary use of fully qualified name 'TestDataUtils.getNewsLikeContentType' due to existing static import 'com.dotcms.datagen.TestDataUtils.getNewsLikeContentType'](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=6105922)